docs/api-guide/caching.md

```diff
@@ -28,34 +28,31 @@ from rest_framework import viewsets
 
 class UserViewSet(viewsets.ViewSet):
     # With cookie: cache requested url for each user for 2 hours
-    @method_decorator(cache_page(60*60*2))
+    @method_decorator(cache_page(60 * 60 * 2))
     @method_decorator(vary_on_cookie)
     def list(self, request, format=None):
-        content = {
-            'user_feed': request.user.get_user_feed()
-        }
+        content = {"user_feed": request.user.get_user_feed()}
         return Response(content)
 
 
 class ProfileView(APIView):
     # With auth: cache requested url for each user for 2 hours
-    @method_decorator(cache_page(60*60*2))
-    @method_decorator(vary_on_headers("Authorization",))
+    @method_decorator(cache_page(60 * 60 * 2))
+    @method_decorator(
+        vary_on_headers(
+            "Authorization",
+        )
+    )
     def get(self, request, format=None):
-        content = {
-            'user_feed': request.user.get_user_feed()
-        }
+        content = {"user_feed": request.user.get_user_feed()}
         return Response(content)
 
 
 class PostView(APIView):
     # Cache page for the requested url
-    @method_decorator(cache_page(60*60*2))
+    @method_decorator(cache_page(60 * 60 * 2))
     def get(self, request, format=None):
-        content = {
-            'title': 'Post title',
-            'body': 'Post content'
-        }
```

Comment:
```python
content = {
            'title': 'Post title',
            'body': 'Post content'
        }
```

Reply:
This is something changed by the command, not a manual change.